recipes/r-gbj/meta.yaml

```
@@ -0,0 +1,93 @@
+{% set version = '0.5.3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-gbj
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/GBJ_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/GBJ/GBJ_{{ version }}.tar.gz
+  sha256: 100b3ba8aff707212dc24872d3e15886adb0382a01cbed848567fcce666b9a8a
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-bh
+    - r-rcpp >=0.12.7
+    - r-skat
+    - r-mvtnorm
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-bh
```

```suggestion
    #- r-bh
```
It's a compile-time dependency that can be commented out here to reduce dependencies.